srcpkgs/strawberry/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'strawberry'
+pkgname=strawberry
+version=0.4.2
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config qt5-host-tools qt5-devel protobuf-c"
+# maybe also liblastfm, but that depends on qt4 right now
+makedepends="chromaprint-devel boost-devel gst-plugins-base1-devel
```

Comment:
This can be reduced a little. 

    makedepends="alsa-lib-devel boost-devel chromaprint-devel gst-plugins-base1-devel
     libcdio-devel libgpod-devel libmtp-devel protobuf-devel pulseaudio-devel
     qt5-plugin-mysql qt5-plugin-odbc qt5-plugin-pgsql qt5-plugin-sqlite
     qt5-plugin-tds qt5-x11extras-devel taglib-devel"